app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MessagesController < ApplicationController
+
+  def create
+    @message = Message.new(message_params)
+    @match = Match.find_by(id: params[:match_id])
+    @message.user = current_user
+    @message.match = @match
+    @my_item = current_user.items.first
```

Comment:
Is this safe? You aren't imposing a specific order on the items collection. It might be worth putting an explicit order by on the has_many